Dockerfile.amd64

```diff
@@ -1,4 +1,4 @@
-FROM node:8-alpine
+FROM node:15.5.1-alpine3.10
```

Comment:
>15.5.1-alpine3.10 [](start = 10, length = 17)

Initially we were using a fixed node major version, why not continue with this approach? For example this image [https://hub.docker.com/layers/amd64/node/15-alpine/images/sha256-7d8fcef4cc84dafe2f4fbcbcd58ee704d452f1e49bc3393c9452992713808dbc?context=explore](https://hub.docker.com/layers/amd64/node/15-alpine/images/sha256-7d8fcef4cc84dafe2f4fbcbcd58ee704d452f1e49bc3393c9452992713808dbc?context=explore)